android/app/src/main/java/org/chrome/device/ml/experiments/MobileBertExperiment.java

```diff
@@ -0,0 +1,115 @@
+/* Copyright 2020. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+package org.chrome.device.ml.experiments;
+
+import android.content.Context;
+import android.os.Handler;
+import android.os.HandlerThread;
+import android.os.Message;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.chrome.device.ml.thirdparty.ml.LoadDatasetClient;
+import org.chrome.device.ml.thirdparty.ml.QaAnswer;
+import org.chrome.device.ml.thirdparty.ml.QaClient;
+
+/**
+ * Mobilebert Question/Answer Experiment
+ */
+public class MobileBertExperiment implements Experiment {
+    private static final String TAG = "MobileBertExperiment";
+    private static final String MODEL_PATH = "bert_model.tflite";
+
+    private final Context context;
+    private Handler handler;
+    private Handler UIHandler;
+
+    private LoadDatasetClient datasetClient;
+    private QaClient qaClient;
+    private ArrayList<Double> timing;
```

Comment:
prefer for variables to just use private List<Double> rather than impl of list - you can instantiate as ArrayList later

Reply:
Changed it to: private List<Double> timing;